service/service.go

```diff
@@ -499,46 +539,35 @@ func (t *service) GetRecord(ctx context.Context, id thread.ID, rid cid.Cid) (cor
 	return cbor.GetRecord(ctx, t, rid, fk)
 }
 
-// record wraps a core.Record within a thread and log context.
-type record struct {
+// Record wraps a core.Record within a thread and log context.
+type Record struct {
 	core.Record
 	threadID thread.ID
 	logID    peer.ID
 }
 
+// NewRecord returns a record with the given values.
+func NewRecord(r core.Record, id thread.ID, lid peer.ID) core.ThreadRecord {
```

Comment:
Yes, nice to have this as part of the public API (I mean its needed, but its also nice)